src/vector/values/values.yaml

```diff
@@ -5,6 +5,10 @@
 role: "Agent"
 
 customConfig:
+  api:
+    enabled: true
+    address: 0.0.0.0:8686
+    playground: false
```

Comment:
Do we still want this enabled by default? I don't think it is being used by the tests anymore?

Reply:
Agree, let's disable it.